packages/framework/src/Resources/styles/admin/components/in/icon.less

```diff
@@ -28,8 +28,9 @@
         top: 2px;
     }
 
-    &--info {
```

Comment:
this might be a BC break because you are "renaming" a class and that can be done only in Major version said by our [BC promise](https://github.com/shopsys/shopsys/blob/master/docs/contributing/backward-compatibility-promise.md#less--css)

Reply:
It is not renaming - I moved colour little bit down - there were dublicated class name --info {}
And added new one.